build.yaml

```diff
@@ -63,16 +86,25 @@ steps:
     - batch_pods_ns
  - kind: buildImage
    name: base_image
-   dockerFile: docker/Dockerfile.base
-   contextPath: .
+   dockerFile: /io/repo/docker/Dockerfile.base
+   contextPath: /io/repo
    publishAs: base
+   inputs:
+    - from: /repo
+      to: /io/repo
+   dependsOn:
+     - clone
  - kind: buildImage
    name: service_base_image
-   dockerFile: docker/Dockerfile.service-base
-   contextPath: .
+   dockerFile: /io/repo/docker/Dockerfile.service-base
+   contextPath: /io/repo
    publishAs: service-base
    dependsOn:
     - base_image
+    - clone
+   inputs:
+    - from: /repo
+      to: /io/repo
```

Comment:
This one needs hailtop, web_common and gear.  Worth it?  I feel like base and service-base are on the critical paths so probably, and other images maybe less sensitive.